validation_components/validation.py

```diff
@@ -6,23 +6,64 @@ def validation_runner(arguments: argparse.Namespace):
     loader = SpreadsheetLoader(arguments.spreadsheet)
     all_entries = loader.load()
 
+    error_list = verify_entries(all_entries)
+
+    if len(error_list) > 0:
+        print('Errors found within manifest:\n\t' + '\n\t'.join(error_list) + '\nPlease correct mistakes and validate again.')
+    else:
+        print('Manifest successfully validated, no errors found!')
+
+
+def verify_entries(all_entries):
     error_list = []
-    registered_values = set()
+    registered_values = {'__null____null__': [1, None, None]}
```

Comment:
Okay I see what you're doing here - you don't want to recreate the "statements" every time, and I think that this is a good idea. But using an array is clumsy. If such thing was needed at all, I'd replace it by an object of a new class, but I don't think you need it at all. Why not just create a dictionary with your hash as key and the actual error message as value? That is easily reusable, only need to attach the sample name to it before adding it to your list.
Valid keys could have value 'None'
I also don't understand the reason for the initialization.